uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/WorkDay.php

```diff
@@ -143,13 +133,15 @@ private static function validateWorkHours($params) {
     $typeOfDay = empty($params['type']) ? null : $params['type'];
     $timeFrom = empty($params['time_from']) ? null : $params['time_from'];
     $timeTo = empty($params['time_to']) ? null : $params['time_to'];
-    $break = empty($params['break']) ? null : $params['break'];
-
-    $hasTimesOrBreak = $timeFrom || $timeTo || $break;
+    $break = isset($params['break']) && $params['break'] == '0' ? 0 :
+      (empty($params['break']) ? null : $params['break']);
```

Comment:
`$break = !$breakIsNotEmpty ? null : $params['break'];` that will throw a notice since `$params['break']` is not really defined in PHP in case of 0. But I **do** get your point, will improve.

Reply:
This has already taken care of it `isset($params['break']) && $params['break'] == 0)`. It checks that break exists and is zero.

Reply:
@tunbola please take my apologies, it works actually, I have no idea why it didn't earlier when I tested it.